packages/cli-config/src/hooks/init.js

```diff
@@ -1,7 +1,8 @@
 import PercyConfig from '@percy/config';
-import { schema } from '@percy/core/dist/config';
+import { schema, migration } from '@percy/core/dist/config';
 
-// ensures the core schema is loaded
+// ensures the core schema and migration is loaded
 export default function() {
   PercyConfig.addSchema(schema);
+  PercyConfig.addMigration(migration);
 }
```

Comment:
We could technically move this to `@percy/cli-exec` where it might be technically more appropriate? 🤔

Reply:
Hmm, I don't feel strongly either way -- your call 👀